crates/hyperqueue/src/dashboard/data/mod.rs

```diff
@@ -55,6 +55,18 @@ impl DashboardData {
         self.worker_timeline.get_connected_worker_ids(time)
     }
 
+    pub fn query_worker_cpu_util(
```

Comment:
This query is a bit too "low-level" I think. We should have a query for getting the worker overview for a specific worker at some time. Getting utilization (or any other) information from the overview should be up to the caller, we probably won't split the state of a single overview inside the query provider.